lib/endpoints/class-wp-json-post-types-controller.php

```diff
@@ -3,6 +3,33 @@
 class WP_JSON_Post_Types_Controller extends WP_JSON_Controller {
 
 	/**
+	 * Register the routes for the objects of the controller
+	 * 
+	 */
+	public function register_routes() {
+
+		parent::register_routes();
```

Comment:
Let's remove this unless it's needed. The base controller shouldn't need to register endpoints, right?